### DISCUSSION TITLE
Reference the same table twice in a query?

### DISCUSSION THREAD

**Discussion by OliverGilan** [prisma/prisma#8491] [Q&A] [answered]
Hello!

I am using Prisma for a new project and I really like it so far but I've run into a scenario I'm not sure how to approach. I am creating a website with products in various categories. These categories can have arbitrary nesting so some categories can be 3 levels deep, some 2 levels, some no levels, etc. I'm using the Nested Set model (http://mikehillyer.com/articles/managing-hierarchical-data-in-mysql/)  to model this heirarchy in the db and I have a scenario where I want to query all the parent categories for a specific category. The specific query in sql is as follows: 

```sql
SELECT parent.name
FROM nested_category AS node,
        nested_category AS parent
WHERE node.lft BETWEEN parent.lft AND parent.rgt
        AND node.name = 'FLASH'
ORDER BY parent.lft;
```
Is there a way to model this query in prisma? Right now I'm using $queryRaw but if there's a better way to do it I'd love to know!

**Comment by ryands17** [comment, marked as answer]
@OliverGilan 👋 

You can use `include` to fetch the same relation and the depth would be defined by how many times you use `include`. This will give you the categories and subcategories.

We have a request to make this easier so it would be great if you could add a 👍 to this, but for now the above way is how you would do it with Prisma.
https://github.com/prisma/prisma/issues/4562

**Reply by OliverGilan** [reply]
That issue describes my problem perfectly. Thumbed up

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
